src/opm/output/eclipse/Summary.cpp

```diff
@@ -3007,6 +3194,24 @@ namespace Evaluator {
         return true;
     }
 
+    bool Factory::isInterRegionValue()
+    {
+        auto keyword = this->node_->keyword;
+
+        auto end = keyword.find('_');
+        if (end == std::string::npos)
+            end = std::min(keyword.size(), std::string::size_type{5});
+
+        auto pos = interregion_units.find(keyword.substr(0, end));
+        if (pos == interregion_units.end())
+            return false;
+
+        // 'node_' represents an inter-region value.  Capture unit of
+        // measure and return true.
+        this->paramUnit_ = pos->second;
+        return true;
```

Comment:
Could this be done by a regex. Too late for me to answer.

Reply:
> Could this be done by a regex. Too late for me to answer.

Possibly, but not easily given the number of special cases.

Reply:
Seems like this is used to not allow "_" in the extension here (for some cases, but maybe not for all)
- R[OGW]F[TR][-+][A-Z0-9_]{3}: take ROFT-A_A, we get ROFT-A

Might not be directly obvious to everybody. Maybe add some documentation?

Reply:
> Seems like this is used to not allow "_" in the extension here

I was really trying to reduce the keyword string down to its "canonical" form, i.e., to remove all traces of a user-defined region name which enables comparing the keyword to the explicit list.  The above didn't really cover all cases however, so I've corrected this code in the latest update.

Reply:
:sunglasses: